ietf/meeting/tests_models.py

```diff
@@ -158,7 +158,7 @@ def test_alpha_str(self):
 
     def test_session_recording_url(self):
         group_acronym = "foobar"
-        meeting_date = datetime.date.today()
+        meeting_date = datetime_today()
```

Comment:
This is a `datetime` - you probably want `date_today()`

Reply:
Changed to `date_today()`.